### PR TITLE
Support the `smart_rollup_execute_outbox_message` in the Contract API

### PR DESCRIPTION
**Pull request by skubarenko**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [x] Your code builds cleanly without any errors or warnings
- [x] You have run the linter against the changes
- [x] You have added unit tests (if relevant/appropriate)
- [ ] You have added integration tests (if relevant/appropriate)
- [x] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [x] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

In this PR, please also make sure: 

- [ ] You have linked this PR to the issue by putting `closes #TICKETNUMBER` in the description box (when applicable)
- [x] You have added a concise description on your changes
## Release Note Draft Snippet
- `@taquito/taquito` Support the `smart_rollup_execute_outbox_message` operation in the Contract API #2321  

---

## About Integration Tests

Hello 👋🏻

I attempted to write integration tests but encountered the following issues:
1. We require a deployed special rollup with a kernel to create outbox messages. Currently, I don't have such a rollup.
2. We need to receive the dynamic commitment hash and output proof for outbox message execution. To obtain these parameters, we need the persistent rollup node URL. We can't use hardcoded commitment hash and output proof since executing the outbox message is possible only once.

As a result, I skipped integration tests for the `smart_rollup_execute_outbox_message` operation.

However, I tested it locally for the Etherlink Bridge on Nairobinet: [onyjJoL7TeSLy1AjWuLQVRqTWVc7pRhq4VhDyMQXXMTjuv7pEtV](https://nairobinet.tzkt.io/onyjJoL7TeSLy1AjWuLQVRqTWVc7pRhq4VhDyMQXXMTjuv7pEtV/862810)

I suggest skipping the integration test for this PR 🙂. Perhaps you can write it later in a separate PR.

### CODE REVIEW

File: integration-tests/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "test:weeklynet": "WEEKLYNET=true jest",
     "test:weeklynet-secret-key": "RUN_WEEKLYNET_WITH_SECRET_KEY=true jest --runInBand",
     "test:ghostnet": "GHOSTNET=true jest",
-    "test:ghostnet-secret-key": "RUN_GHSOTNET_WITH_SECRET_KEY=true jest --runInBand",
+    "test:ghostnet-secret-key": "RUN_GHOSTNET_WITH_SECRET_KEY=true jest --runInBand",
```
Comment (hui-an-yang) [inline integration-tests/package.json:11]: thanks for fixing typo

File: packages/taquito/src/contract/interface.ts [hunk 2]
```diff
@@ -273,6 +275,16 @@ export interface ContractProvider extends StorageProvider {
    * @param SmartRollupOriginateParams smartRollupOriginate operation parameter
    */
   smartRollupOriginate(params: SmartRollupOriginateParams): Promise<SmartRollupOriginateOperation>;
+  /**
+   * @description Execute an outbox message of the specified smart rollup
```
Comment (hui-an-yang) [inline packages/taquito/src/contract/interface.ts:279]: slightly nitpick would be great to use  octez-client description `Execute a message from a smart rollup's outbox of a cemented commitment`. Also applies to other files

File: docs/smart_rollups.md
Comment (hui-an-yang) [inline docs/smart_rollups.md:1]: Can you also updated changes of this file to `website/versioned_docs/version-19.0.0/smart_rollups.md` too?

**Review by hui-an-yang — COMMENTED** [review]
💫 Great work, only have minor comments, thank you so much! 
I also changed the branch to merge into so that we can see website preview before merging in master. Will wait for Davis's review too

File: docs/smart_rollups.md
Comment (skubarenko) [reply docs/smart_rollups.md:1]: Done!

**Pull request merged**